Jellyfin.ApiClient/ApiWebSocket.cs

```diff
@@ -318,14 +279,14 @@ private void DisposeWebSocket()
         /// </summary>
         /// <param name="serverAddress">The server address.</param>
         /// <returns>System.String.</returns>
-        protected string GetWebSocketUrl(string serverAddress)
+        protected Uri GetWebSocketUrl(Uri serverAddress)
         {
             if (string.IsNullOrWhiteSpace(AccessToken))
             {
                 throw new ArgumentException("Cannot open web socket without an access token.");
             }
 
-            return serverAddress.Replace("http:", "ws:").Replace("https:", "wss:") + "/embywebsocket?api_key=" + AccessToken + "&deviceId=" + DeviceId;
+            return new Uri(serverAddress.ToString().Replace("http:", "ws:").Replace("https:", "wss:") + "/embywebsocket?api_key=" + AccessToken + "&deviceId=" + DeviceId);
```

Comment:
We migrated to `/socket` for a more generic endpoint.

Reply:
Done in 3641a54.